triggerbot/tree_watcher.py

```diff
@@ -137,7 +137,10 @@ def requested_trigger(self, branch, rev, builder):
                 return
 
             seen_builders.add(builder)
-            count = self.revmap[rev]['requested_trigger']
+            count = self.revmap[rev]['requested_trigger'][0]
+            if self.revmap[rev]['requested_trigger'][1] != 0 and 'talos' in builder:
```

Comment:
Python allows python matching assigment for tuples, so this could be:

```python
count, talos_count = self.revmap[rev]['requested_trigger']
if talos_count and 'talos' in builder:
    count = talos_count
```